### PR TITLE
[16.0][mig] default_multi_user: Migration to 16.0

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

**Review by kikopeiro — APPROVED** [review]
Functional review / no code review. :+1:  thanks !

File: default_multi_user/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Lois Rilo <lois.rilo@forgeflow.com>
+* Sunanda Chhatbar <sunanda.chhatbar@initos.com>
+* Dhara Solanki <dhara.solanki@initos.com>
```
Comment (CRogos) [inline default_multi_user/readme/CONTRIBUTORS.rst:3]: As fare as I know are migrations without any code change not enough to put you on the contributor list.

**Review by CRogos — COMMENTED** [review]
The marked commits can be squashed.
image
see also: https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate

File: default_multi_user/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (kikopeiro) [reply default_multi_user/readme/CONTRIBUTORS.rst:3]: Please @dsolanki-initos could you review this PR?

**Pull request closed without merging**